src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-linkedit.js

```diff
@@ -386,14 +393,35 @@ export default class EzBtnLinkEdit extends Component {
             linkAttrs.href = this.state.linkHref;
             linkUtils.update(linkAttrs, this.state.element, modifySelection);
 
-            editor.fire('actionPerformed', this);
+            this.restoreScrollbarPosition(() => {
+                editor.fire('actionPerformed', this);
+            });
         }
 
         // We need to cancelExclusive with the bound parameters in case the
         // button is used inside another component in exclusive mode (such
         // is the case of the link button)
         this.props.cancelExclusive();
     }
+
+    /**
+     * Saves current scrollbar position, invokes callback function and scrolls
+     * to the saved position afterward.
+     *
+     * @method restoreScrollbarPosition
+     * @param {Function} callback invoked after saving current scrollbar position
+     */
+    restoreScrollbarPosition(callback) {
```

Comment:
I don't like the name of this method. Not really `restore`. 
Maybe something like `invokeWithFixedScrollbar`?